cmd/relayproxy/service/gofeatureflag.go

```diff
@@ -141,6 +142,14 @@ func initRetriever(c *config.RetrieverConf) (retriever.Retriever, error) {
 			RepositorySlug: c.RepositorySlug,
 			Timeout:        retrieverTimeout,
 		}, nil
+	case config.BitbucketRetriever:
+		return &bitbucketretriever.Retriever{
```

Comment:
**issue:** We are missing the field `Commit` _(`Branch` if we are using the new name)_, in the creation of the retriever for the relay proxy.